.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
It's worth looking at the diff before you open a PR. This needs to be in the .gitignore and removed.